### PR TITLE
feat: Deckbuilder

### PR DESCRIPTION
**Pull request by JoaquinLeonArg**

### CODE REVIEW

File: frontend/src/app/[tournamentID]/decks/[deckID]/export.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  let text = ""
+  deck.cards
+    .filter((card) => card.board == "b_mainboard")
+    .forEach(
+      (card) => {
+        text += `${card.count} ${cardsById[card.owned_card_id].card_data.name}\n`
+      }
+    )
+  text += "Sideboard\n"
```
Comment (fbianco32) [inline frontend/src/app/[tournamentID]/decks/[deckID]/export.ts]: Cockatrice takes \n as separator for sideboard, this will add an unknown card

**Comment by fbianco32** [comment]
Deck list screen does not have a loading spinner

**Comment by fbianco32** [comment]
image
Says deck name twice

**Comment by fbianco32** [comment]
image
Maybe add a [+] button to add cards instead of just clicking it? Like moxfield does. May be more intuitive

**Comment by fbianco32** [comment]
image
Make [X] bigger and red maybe? Didnt see it first time around

**Review by JoaquinLeonArg — COMMENTED** [review]
@fbianco32 ready for a new review. I've removed the close buttons from the modals since they already have a close button at the bottom.

**Review by fbianco32 — APPROVED** [review]

**Review by fbianco32 — APPROVED** [review]

**Pull request merged**